sql-cli/conftest.py

```diff
@@ -161,3 +164,15 @@ def initialised_project_with_empty_workflow(initialised_project: Project):
         dst=initialised_project.directory / "workflows" / "empty",
     )
     return initialised_project
+
+
+@pytest.fixture()
+def test_connections():
+
+    merge_conn(
+        Connection(
+            conn_id="my_test_sqlite",
+            conn_type="sqlite",
+            host=os.path.join(gettempdir(), "sqlite_default.db"),
```

Comment:
Do we need this path to be "static" ?

If not, I would suggest we use pytests `tmp_path` fixture:
```suggestion
            host=(tmp_path / "sqlite_default.db").as_posix(),
```